### PR TITLE
Migrate a brief repository creation guide from [organization]

### PR DESCRIPTION
**Pull request by 0x2b3bfa0**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,33 @@
-# DevOps
+# Helpful Engineers GitHub Organization
```
Comment (0x2b3bfa0) [inline README.md:1]: The title should be changed to «DevOps team» or something along these lines.
Preferably, including a link to the Slack channel.

File: README.md [hunk 2]
```diff
@@ -1,3 +1,33 @@
...
+>
+> **Guides, instructional videos, and other materials**  
+>
+> Accompanying materials such as instruction manuals, videos and other copyrightable works that are useful for but not necessary to design, make, test or prepare the Medical Hardware for distribution should be published under a [Creative Commons Attribution 4.0](https://slack-redir.net/link?url=https%3A%2F%2Fcreativecommons.org%2Flicenses%2Fby%2F4.0%2F&v=3) License. A reference to this license should be included at the bottom of vital documents, and we recommend the following:  
+>
+> > _All copyrightable materials  other than software, and everything needed to design, make, test and prepare the hardware for distribution, such as instructional videos and manuals, are published under a_ [Creative Commons Attribution 4.0](https://slack-redir.net/link?url=https%3A%2F%2Fcreativecommons.org%2Flicenses%2Fby%2F4.0%2F&v=3) _License._
+>
+> As a follow up, we will be asking all contributors to acknowledge that their designs are open source.
+
+**Source: [https://helpfulengineering.slack.com/archives/C010GJ68R1N/p1584974903050100](https://helpfulengineering.slack.com/archives/C010GJ68R1N/p1584974903050100)**
```
Comment (0x2b3bfa0) [inline README.md:33]: This pull request should be merged with #2

**Review by dmangot — CHANGES_REQUESTED** [review]
Please move this to a file about repositories, not in the main README for our entire team.  We do a lot more than repository creation which is what putting this at the level of the entire team suggests.

File: README.md (hunk 1 above)
Comment (dmangot) [reply README.md:1]: 👍

**Comment by 0x2b3bfa0** [comment]
Closing in favor of #4

**Pull request closed without merging**